app/package.json

```diff
@@ -60,6 +61,8 @@
     "objection": "^2.2.0",
     "openid-client": "^3.15.2",
     "pg": "^8.2.1",
+    "prompt-sync": "^4.2.0",
+    "readline-sync": "^1.4.10",
```

Comment:
Given that this is solely used for the migration and it's supposed to be managed by a developer, this could be a devDependency instead.
Also not sure from just code level view if we're actually using prompt-sync - I only see references of it in package and package-lock.